go/cmd/dolt/commands/sql.go

```diff
@@ -1002,30 +983,18 @@ func runShell(ctx context.Context, se *engine.SqlEngine, mrEnv *env.MultiRepoEnv
 		query = strings.TrimSuffix(query, shell.LineTerminator())
 		resultFormat := se.GetResultFormat()
 
-		// TODO: it would be better to build this into the statement parser rahter than special case it here
+		// TODO: it would be better to build this into the statement parser rather than special case it here
 		for _, terminator := range verticalOutputLineTerminators {
 			if strings.HasSuffix(query, "\\G") {
```

Comment:
This is a bug: it should use `terminator` instead of `"\\G"` here

As a result, `\G` works but `\g` does not

Reply:
Ohhh I see, that makes sense! I’ll fix this